corehq/apps/locations/models.py

```diff
@@ -543,6 +544,7 @@ def location_type(self):
         return self.location_type_object.name
 
     _sql_location_type = None
+
     @location_type.setter
     def location_type(self, value):
```

Comment:
I'm not a huge fan of 100% enforcing the rule that there must always be a blank line above a function definition.  The two in this file, for instance, I think are exceptions.

The function on line 63 here mutates `ordered_loc_types` - there's only a function because that traversal is easier done through recursion.  Conceptually, it's all part of one block in the same way a `for` loop would be.

This one inits a private variable that's analogous to the public property, which is why they're similarly grouped.